### DISCUSSION TITLE
Is there a regular sync meeting for k3s development?

### DISCUSSION THREAD

**Discussion by harsimranmaan** [k3s-io/k3s#8836] [General]
Some CNCF project have bi-weekly/monthly sync meetings/office hours for the community. Are there any for k3s? My main motivation is to find a way to discuss Embedded registry support as outlined in the roadmap https://github.com/k3s-io/k3s/blob/master/ROADMAP.md

**Comment by brandond** [comment]
Not at this time, no. We have been meaning to do that for a while. When we do, we will make sure to have a representative from our product management team present.

**Reply by brandond** [reply]
For what it's worth, there were various attempts at embedding distributed registry support, along side a distributed container build system, attempted a year or two ago. Whatever we do needs to be
1. Lightweight
2. Highly available
3. Distributed
4. Integrated into the existing K3s management of the containerd configuration
6. Not change existing k3s behavior for administrators that do not want to use it
